staging/src/k8s.io/component-base/config/testing/helpers.go

```diff
@@ -70,6 +71,10 @@ func RunTestsOnYAMLData(t *testing.T, scheme *runtime.Scheme, tests []TestCase,
 			if needsUpdate {
 				const updateEnvVar = "UPDATE_COMPONENTCONFIG_FIXTURE_DATA"
 				if os.Getenv(updateEnvVar) == "true" {
+					if err := os.MkdirAll(filepath.Dir(rt.out), 0755); err != nil {
```

Comment:
hm, why did this work before?

Reply:
the directory was already present on my machine, so it didn't complain :(